Pyjion/intrins.cpp

```diff
@@ -1294,6 +1228,44 @@ int PyJit_DeleteGlobal(PyFrameObject* f, PyObject* name) {
     return PyDict_DelItem(f->f_globals, name);
 }
 
+PyObject *
+_PyDict_LoadGlobal(PyDictObject *globals, PyDictObject *builtins, PyObject *key)
+{
+#if FALSE
```

Comment:
It's the ideal way to do the lookup...  CPython is avoiding hashing the key twice and is going straight to the internal data structure to do the lookup. We used to do the same thing, but it required some patches.  Now we're just using the available public API surface.  It'd be nice to bring it back, but I'll get rid of the commented out code